src/Sarif.WorkItems/SarifWorkItemFiler.cs

```diff
@@ -38,6 +42,8 @@ public SarifWorkItemFiler(Uri filingUri = null, SarifWorkItemContext filingConte
             }
 
             this.FilingClient = FilingClientFactory.Create(this.FilingContext.HostUri);
```

Comment:
>FilingClientFactory.Create [](start = 32, length = 26)

In future, we could have an IFilingClientFactory service. This might be useful in helping to generalize a mocking story. Probably not required now, just would introduce another interface type...

Reply:
That's a good idea. I've noticed a few places where services could be injected. 

---
In reply to: [396835947](https://github.com/microsoft/sarif-sdk/pull/1820#discussion_r396835947) [](ancestors = 396835947)